setup.py

```diff
@@ -103,7 +102,7 @@ def input_password(self, key, prompt, hasher=None, required=False):
         configured = self.data.get(key, u'')
         prompt = u'\n{} [{}]: '.format(prompt, u'*****' if configured else u'')
         while True:
-            inputed = getpass.getpass(PROMPT + prompt + RESET)
+            inputed = raw_input(PROMPT + prompt + RESET)
```

Comment:
Naco si tej `getpass.getpass` menil na `raw_input` vlastne? Skusil som to zmenit naspat a setup normalne zbehol. Vid check na branchi `tmp`: https://github.com/martinmacko47/chcemvediet/runs/819330815?check_suite_focus=true

Reply:
:ok: Ked som testoval `getpass.getpass` v terminaly lokalne na mojom zariadeni, tak mi nezobralo. Vobec mi nenapadlo ze, github to zoberie.